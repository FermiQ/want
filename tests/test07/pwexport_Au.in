&inputpp
  prefix='gold'
  outdir='./'
  pseudo_dir = './'
  psfile(1) = 'Au11pw91.mt.UPF'
  single_file = .FALSE.
  ascii = .FALSE.
/
