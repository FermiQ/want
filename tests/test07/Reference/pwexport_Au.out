
     Program POST-PROC v.3.1    starts ...
     Today is 12Apr2006 at  0:18:57 

     nbndx  =    20  nbnd   =    20  natomwfc =    18  npwx   =    8007
     nelec  =  22.00  nkb   =    16  ngl    =    3621

     Check: negative/imaginary core charge=   -0.000071    0.000000
